src/platforms/vue-native/scripts/compiler.js

```diff
@@ -248,3 +271,46 @@ function traverse(ast, nodes = []) {
     })
   }
 }
+
+function getFirstLineOfText(content, text) {
+  var firstLine = 0
+  let splittedContent = content.split(newLine)
+  if (!content.includes(text)) {
+    return splittedContent.length
+  }
+  splittedContent.some((line, index) => {
+    if (line.includes(text)) {
+      firstLine = index
+      return true
+    } else {
+      return false
+    }
+  })
+  return firstLine
+}
+
+function getFirstNonLineOfText(content, text) {
+  var firstNonCommentedLine = 0
```

Comment:
```suggestion
  let firstNonCommentedLine = 0
```

Use `const` for declarations and `let` only if there is a need to mutate. Avoid `var`